src/test/java/org/jamesframework/core/search/NeighbourhoodSearchTest.java

```diff
@@ -74,7 +74,47 @@ public static void setUpClass() {
     public static void tearDownClass() {
         System.out.println("# Done testing NeighbourhoodSearch!");
     }
-    
+
+    @Test
+    public void testGetBestMoveAcceptFirstImprovement() throws Exception {
```

Comment:
Please move test method down (alongside other test methods instead of in between the setup code).